good-enough-practices-for-scientific-computing.tex

```diff
@@ -1129,8 +1131,7 @@ \section{What We Left Out}\label{sec:omitted}
 \item[\textbf{Coverage}] Every modern programming language comes with
   tools to report the \emph{coverage} of a set of test cases, i.e.,
   the set of lines that are and aren't actually executed when those
-  tests are run. Mature projects run these tools periodically to find
-  code that isn't being used any more, but as with unit testing, this
+  tests are run. But as with unit testing, this
```

Comment:
Removed wording about finding code that's not being used as technically tests can't really reveal that.